src/plugins/plugin-express.ts

```diff
@@ -55,7 +55,7 @@ function patchModuleRoot(express: Express4Module, api: PluginTypes.TraceAgent) {
       api.wrapEmitter(req);
       api.wrapEmitter(res);
 
-      const url = `${req.protocol}://${req.hostname}${req.originalUrl}`;
+      const url = `${req.protocol}://${req.headers.host}${req.originalUrl}`;
```

Comment:
Curious. What's the diff between `req.hostname` and `req.headers.host`?

Reply:
I believe `headers.host` includes the port number, but `hostname` doesn't.